deploy/Dockerfile

```diff
@@ -122,19 +121,6 @@ RUN cd /tmp && curl -fsSL -o helm.tar.gz "${HELM3_URL}" \
   && ln -s "${KOTS_HELM_BIN_DIR}/helm3" "${KOTS_HELM_BIN_DIR}/helm" \
   && rm -rf helm.tar.gz linux-amd64
 
-# Install s3cmd
-ENV S3CMD_VERSION=2.1.0
```

Comment:
Wondering if it had to do with the `python-dateutil` dependency? https://github.com/replicatedhq/kots/pull/3420/files#diff-5b6228cfe643850b0b1c8d2b5d69430bb121272e145b651c2ff6a85c23895b4eL3.

Since it's included in the Bullseye packages now, we can install with `apt` and the dependencies appear to be getting installed properly: https://packages.debian.org/bullseye/utils/s3cmd

Reply:
ah, that might be right.